internal/ingress/types.go

```diff
@@ -292,6 +295,10 @@ type Location struct {
 	// addresses or networks are allowed.
 	// +optional
 	Whitelist ipwhitelist.SourceRange `json:"whitelist,omitempty"`
+	// Blocklist indicates connections from certain client
+	// addresses or networks will be rejected..
+	// +optional
+	Blocklist ipblocklist.SourceRange `json:"blocklist,omitempty"`
```

Comment:
Not now, but in a future I guess we can consolidate the SourceRange as some unique type, and return based on the annotation